ctakesclient/filesystem.py

```diff
@@ -216,6 +216,10 @@ def read_json(filename) -> dict:
     with open(filename, 'r', encoding='utf-8') as fp:
         return json.load(fp)
 
+def write_json(filename, data: dict) -> None:
```

Comment:
If you make the unit test change I suggest, you can remove this method again as nothing will be using it.